src/main/java/com/cegeka/academy/web/rest/ChallengeController.java

```diff
@@ -30,6 +31,11 @@ public void delete(@PathVariable long id) throws NotFoundException {
         challengeService.deleteChallenge(id);
     }
 
+    @RequestMapping(value = "/update", method = RequestMethod.PUT)
```

Comment:
Orice metoda de REST care se adreseaza unei singure resurse, trebuie sa contina in path id-ul. In cazul tau, doresti sa actualizezi o resursa de tip Challenge, ar trebui sa primesti si id-ul in path. La fel se intampla pentru GET sau DELETE in cazul unei singure resurse